packages/components/src/components/Text/Text.stories.tsx

```diff
@@ -24,10 +24,14 @@ const meta: Meta<TextProps> = {
   parameters: {
     docs: generateDocs({
       name: 'Text',
-      // docUrl:
-      //   'https://department-of-veterans-affairs.github.io/va-mobile-app/design/Components/Alerts%20and%20Progress/Text',
+      docUrl:
+        'https://department-of-veterans-affairs.github.io/va-mobile-app/design/Components/Typography/Text',
     }),
   },
+  argTypes: {
+    tone: { if: { arg: 'variant', neq: 'display' } },
+    size: { if: { arg: 'variant', neq: 'display' } },
```

Comment:
Might be helpful to document what the `if` and `neq` are doing here. Least I'm not familiar with what this Storybook syntax is doing.

Reply:
Added comments